src/pages/Login/Login.tsx

```diff
@@ -0,0 +1,70 @@
+import { useContext, useState } from 'react';
+import logo from "../../assets/logo.png"
+import { Context } from '../..';
+import { observer } from 'mobx-react-lite';
+import { useLocation, useNavigate } from 'react-router-dom';
+
+function Login() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const { store } = useContext(Context);
+  const [checked, setChecked] = useState(false);
+  const [messageInputFields, setMessageInputFields] = useState("");
+  const navigate = useNavigate();
+  const location = useLocation();
+
+  function handleClick() {
+    const result = store.login(email, password);
+    if (result.error) {
+      setMessageInputFields(result.error);
+    } else {
+      const from = location.state?.from?.pathname || "/";
+      navigate(from, { replace: true });
+    }
+  }
+
+  return (
+    <div className="w-1/4 mt-20 mx-auto">
+      <img src={logo} className="mx-auto size-40 mb-10 rounded-full"></img>
```

Comment:
```suggestion
      <img src={logo} className="mx-auto size-40 mb-10 rounded-full" />
```
плюс нужно добавлять alt аттрибут  к изображениям